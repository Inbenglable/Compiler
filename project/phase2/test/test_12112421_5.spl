int dfs(int x, int f){
    int x = 0;
    int res = 0;
    res = res + dfs(x*2, x);
    res = res + dfs(x*2+1, x);
    return res;
}