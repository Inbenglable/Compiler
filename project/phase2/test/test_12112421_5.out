Error type 3 at line 2: variable "x" is redefined in the same scope
